### PR TITLE
runner: abstract out uwsgi filemon stuff to be more reusable

### PR DESCRIPTION
**Pull request by mattrobenolt**
Brother Feng wants to use this in getsentry.

### CODE REVIEW

File: src/sentry/utils/uwsgi.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+import os
+
+
+def filemon(path):
```
Comment (tkaemming) [inline src/sentry/utils/uwsgi.py]: Without the uWSGI background context, a like `reload_on_file_change` or something would be clearer here, `filemon` is pretty abstract and doesn't really communicate the action being taken here

**Review by tkaemming — APPROVED** [review]
🈶

File: src/sentry/utils/uwsgi.py (hunk 1 above)
Comment (mattrobenolt) [inline src/sentry/utils/uwsgi.py]: Sure, I can rename.

Comment (mattrobenolt) [inline src/sentry/utils/uwsgi.py]: Went with `reload_on_change`.

Comment (tkaemming) [inline src/sentry/utils/uwsgi.py]: :metal:

**Pull request merged**